src/assets/api-calls.tsx

```diff
@@ -12,6 +13,24 @@ export const getApplicants = (): Promise<any> => {
 		})
 }
 
+export const search = 
+	(query: {skills:Array<number>, values: Array<number>}):Promise<any> => {
+		return fetch(`${baseUrl}/applicants/search`, {
+			method: 'POST', 
```

Comment:
Great call on making this a POST instead of a GET!

Reply:
Agreed!